### ISSUE TITLE
Module not found Can't resolve '../../assets/homm3/ui/army_icons.css

### ISSUE THREAD

**Issue by Sajjon** [srg-kostyrko/hoom3-map-viewer#2]
```
/src/components/town/Garrison.tsx
Module not found: Can't resolve '../../assets/homm3/ui/army_icons.css' in '/Users/sajjon/Developer/homm3-map-viewer/src/components/town'
```

I get this when I run `npm start` (after `npm install`), any ideas? Which node version are you using?

**Comment by srg-kostyrko** [comment]
Sorry, I was confident that I added instruction to readme file.

This repository does not have resource files - only scripts to extract them from game resource files.
So before starting - you first need to copy resource archives from installed game  (all `.lod` files in `Data` folder) into `scripts/in` folder and run `npm run extract`. This will generate needed images and css files.
I also noticed that I committed demo map import without map itself - so you'll need to adjust `src/store/AppStore` commenting out `import demoMap from "../assets/homm3/maps/rnd1.json";` and `this.changeMap(demoMap as H3MFile);`
